static/include/sample.md

```diff
@@ -1,24 +1,94 @@
-
 ### This is a Heading in the Sample File
-This entire section was added to the current topic using the readfile shortcode.
+
+This entire section was added to the current topic using the `readfile` shortcode.
 
 {{% alert="Note" color="note" %}}
```

Comment:
```suggestion
{{% alert="Important" color="note" %}}
```
[out of diff nit] While we're here...realizing we don't use "note" anymore so should be consistent through this doc :)